src/loops/while_loops/hard/q6/FactOrIal.java

```diff
@@ -0,0 +1,25 @@
+package loops.while_loops.hard.q6;
+
+/*
+Write a program that will accept a non-negative number from the user and print out the factorial of that number.
+Do so by using a while loop. ( Hint: n factorial or n! = n x (n - 1) x (n - 2) x … x 2 x 1 )
+ */
+import java.util.Scanner;
+public class FactOrIal
```

Comment:
I'm not a fan of the random upper case class name

Reply:
I am not the one that picked the class name, that's just what was in the document. Do you want me to change the document?